src/dparse/ast.d

```diff
@@ -1622,7 +1625,7 @@ final class Foreach(bool declOnly) : BaseNode
     /** */ ForeachType foreachType;
     /** */ Expression low;
     /** */ Expression high;
-    /** */ bool useBrace;
+    /** */ StatementBlockStyle style;
```

Comment:
this should be moved into the `static if (declOnly)` block 2 lines below because in the other block there is already DeclarationOrStatement which can either only be a single thing or a block statement. (where you know it uses braces)

This is only set in the `static if (declOnly)` code block too, which means this would always be "single" otherwise